src/ui/PageHeader/PageHeader.tsx

```diff
@@ -0,0 +1,102 @@
+import { forwardRef } from "react";
+import styled from "styled-components";
+import shapesPink1Url from "./shapes-pink-1.svg";
+import shapesPink2Url from "./shapes-pink-2.svg";
+import shapesBlue1Url from "./shapes-blue-1.svg";
+import { H1, Subheading1, P } from "../Typography";
+
+type BackgroundType = "pink-1" | "pink-2" | "blue-1";
+
+const Wrap = styled.div<{
+  $backgroundType: BackgroundType;
+}>`
+  padding: 80px 0 80px 0;
+  display: flex;
+  flex-wrap: nowrap;
+  flex-direction: row;
+  justify-content: space-evenly;
+  color: var(--color-white);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+
+  ${(props) =>
+    props.$backgroundType === "pink-1" &&
+    `
+    background-image: url(${shapesPink1Url});
+    background-color: var(--color-red);
+  `}
+
+  ${(props) =>
+    props.$backgroundType === "pink-2" &&
+    `
+    background-image: url(${shapesPink2Url});
+    background-color: var(--color-red);
+  `}
+
+  ${(props) =>
+    props.$backgroundType === "blue-1" &&
+    `
+    background-image: url(${shapesBlue1Url});
+    background-color: var(--color-blue);
+  `}
+`;
+
+const BlockHeader = styled.div`
+  flex-grow: 60;
+  flex-basis: 0;
+`;
+
+const BlockSpacer = styled.div`
+  flex-grow: 40;
+  flex-basis: 0;
+`;
+
+const Header = styled.div`
+  padding-left: 18%;
+  padding-right: 2%;
+  display: flex;
+  flex-wrap: nowrap;
+  flex-direction: column;
+  flex-grow: 1;
+  flex-basis: 0;
+`;
+
+const Subheading = styled(Subheading1)`
+  opacity: 0.75;
+  margin: 0;
+`;
+
+const Heading = styled(H1)`
+  margin: 0;
+`;
+
+const Paragraph = styled(P)`
+  padding: 0 12% 0px 0px;
+  line-height: 19.2px;
+  margin: 12px 0 0 0;
+`;
+
+type Props = {
+  markerText: string;
+  heading: string;
+  description: string;
+  backgroundType?: BackgroundType;
+};
+
+export const PageHeader = forwardRef<HTMLDivElement, Props>(
```

Comment:
Do we really need forward the ref here? I can't see it's been used anywhere.

Reply:
It was copy/paste job 🙂  - Although, I don't see the one in Button is used by any caller either?

Reply:
`<Button />` is a bit different. Forward ref proxy the raw DOM `button` reference so that the consumer of `<Button />` component have access to it. This is useful and commonly a good practice to wrap interactive DOM element as React component so that callers can decide to manipulate with the raw element's click, focus, hover, readability events etc. Even though it's not yet used in our project.

Same would apply to wrap other interactive DOM elements like input, textarea, canvas etc.

Reply:
OK yes thanks for that explanation, I can see why it could be useful. I just thought I didn't see it being used currently. But of course if that's what its design is meant to be and just no use cases yet 👍